client/payment-methods/index.js

```diff
@@ -145,10 +141,30 @@ const PaymentMethods = () => {
 	} = useContext( WCPaySettingsContext );
 
 	const { isUpeEnabled, status } = useContext( WcPayUpeContext );
+	const [ useIsSurveyModalOpen, setIsSurveyModalOpen ] = useState( false );
+	const [ useIsDisableModalOpen, setIsDisableModalOpen ] = useState( false );
 
 	return (
 		<>
-			{ 'error' === status && <UpeDisableError /> }
+			{ useIsDisableModalOpen ? (
+				<DisableUPEModal
+					enabledMethods={ enabledMethods }
+					setIsModalOpen={ setIsDisableModalOpen }
+					triggerAfterDisable={ () => setIsSurveyModalOpen( true ) }
+				/>
+			) : null }
+			{ useIsSurveyModalOpen ? (
+				<WcPaySurveyContextProvider>
+					<SurveyModal
+						setIsModalOpen={ setIsSurveyModalOpen }
+						surveyOptions={ {
+							surveyKey: 'wcpay-upe-disable-early-access',
+							surveyQuestion: 'why-disable',
+						} }
```

Comment:
When props are passed as complex values (objects, arrays, etc), it gets harder for React to optimize them, because `{} === {} // false`.

What do you think about extracting this to individual strings?

```
surveyKey="wcpay-upe-disable-early-access"
surveyQuestion="why-disable"
```